### PR TITLE
Add support for Python 3

### PR DESCRIPTION
**Pull request by tsauerwein**
With the help of [2to3](https://docs.python.org/2/library/2to3.html) and some little changes, Python 3 can easily be supported.

### CODE REVIEW

File: pyramid_closure/closure/depswriter.py [hunk 1]
```diff
@@ -30,8 +30,8 @@
 import shlex
 import sys
 
-import source
-import treescan
+from . import source
+from . import treescan
```
Comment (sbrunner) [inline pyramid_closure/closure/depswriter.py]: Personally I don't like relative import... but finally I don't know what we should do ...

Comment (tsauerwein) [inline pyramid_closure/closure/depswriter.py]: Ah sorry, I didn't pay too much attention to the changes 2to3 did. But yes, I also prefer absolute imports, I changed it to:

    from pyramid_closure.closure import source
    from pyramid_closure.closure import treescan

**Comment by tsauerwein** [comment]
Thanks for your review, I updated the PR.

**Comment by sbrunner** [comment]
Thanks @tsauerwein, for me it's ready to merge :-)

**Pull request merged**